src/standalone/Vegetation/photosynthesis.jl

```diff
@@ -207,7 +193,6 @@ function update_photosynthesis!(
     )
     Vcmax25field .= Vcmax25
 end
-Base.broadcastable(m::AbstractPhotosynthesisMechanism) = tuple(m)
```

Comment:
Why was this removed?

Reply:
The AbstractPhotosynthesisMechanism struct definition is deleted above (line 3)